src/aspire/basis/basis.py

```diff
@@ -538,6 +538,9 @@ def expand(self, x):
 
         :param x: An array whose last two or three dimensions are to be expanded
             the desired basis. These dimensions must equal `self.sz`.
+        :param tol: Tolerances for convergence, `norm(residual) <= max(tol*norm(b), atol)`.
```

Comment:
Maybe reword this to `Relative tolerance for convergence` and next one to `Absolute tolerance for convergence`?

Reply:
Sure, sounds good.